frontend/packages/core/src/Input/radio.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import type { RadioProps as MuiRadioProps } from "@material-ui/core";
+import { Radio as MuiRadio } from "@material-ui/core";
+
+const StyledRadio = styled(MuiRadio)(
+  {
+    ".MuiIconButton-label": {
+      height: "24px",
```

Comment:
nit: should we use rem throughout?

Reply:
Daniel and I chatted about this Monday and I think we are going to stick with pixels. There used to be much more of a reason to use rem to prevent scaling issues but I think most modern browsers handle this for us now.

Reply:
ah ok! will update my PR